.github/actions/zombienet-sdk/action.yml

```
@@ -0,0 +1,60 @@
+name: "Zombienet-sdk test"
+inputs:
+  build-id:
+    description: ""
+    required: true
+  ref-slug:
+    description:
+    required: true
+  test:
+    description: "test filter to pass to nextest (e.g: functional::spam_statement_distribution_requests::spam_statement_distribution_requests_test)"
+    required: true
+  prefix:
+    description: "Archive prefix for tests files (e.g polkadot, cumulus or substrate)"
+    required: true
+
+
+runs:
+  using: "composite"
+  steps:
+    - name: k8s_auth
+      shell: bash
+      run: |
+        . /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh
+        k8s_auth
+
+    - name: common_vars
+      shell: bash
+      run: |
+        echo "Vars"
+        echo "ZOMBIENET_INTEGRATION_TEST_IMAGE: $ZOMBIENET_INTEGRATION_TEST_IMAGE"
+        echo "COL_IMAGE: $COL_IMAGE"
+        echo "POLKADOT_IMAGE: $POLKADOT_IMAGE"
+        echo "MALUS_IMAGE: $MALUS_IMAGE"
+        echo "Inputs"
+        echo "test: ${{ inputs.test}}"
+        echo "prefix: ${{ inputs.prefix }}"
```

```suggestion
runs:
  using: "composite"
  env:
    TEST_NAME: ${{ inputs.test }}
    PREFIX: ${{ inputs.prefix }}
  steps:
    - name: k8s_auth
      shell: bash
      run: |
        . /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh
        k8s_auth

    - name: common_vars
      shell: bash
      run: |
        echo "Vars"
        echo "ZOMBIENET_INTEGRATION_TEST_IMAGE: $ZOMBIENET_INTEGRATION_TEST_IMAGE"
        echo "COL_IMAGE: $COL_IMAGE"
        echo "POLKADOT_IMAGE: $POLKADOT_IMAGE"
        echo "MALUS_IMAGE: $MALUS_IMAGE"
        echo "Inputs"
        echo "test: $TEST_NAME"
        echo "prefix: $PREFIX"
```

It's [better to use](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable) an intermediary env variable for all inputs.